core/templates/services/code-normalizer.service.ts

```diff
@@ -69,7 +69,8 @@ export class CodeNormalizerService {
 
       var numSpaces = line.length - this.removeLeadingWhitespace(line).length;
 
-      var existingNumSpaces = Object.keys(numSpacesToDesiredIndentLevel);
+      // eslint-disable-next-line max-len
+      var existingNumSpaces = Object.keys(numSpacesToDesiredIndentLevel).map(Number);
```

Comment:
We prefer to limit the usage of `eslint-disable` exceptions. You can do something like this:
```suggestion
      var existingNumSpaces = Object.keys(
        numSpacesToDesiredIndentLevel
      ).map(Number);
```